docs/usage/satfunc.rst

```diff
@@ -28,16 +28,16 @@ Alternatively, the same data can be produced as a CSV file using the command lin
 
 .. code-block:: console
 
-  ecl2csv satfunc MYDATADECK.DATA --verbose --output satfunc.csv
+  res2csv satfunc MYDATADECK.DATA --verbose --output satfunc.csv
 
 It is possible to extract keywords one at a time using the ``--keywords`` command
 line option.
 
-Instead of Eclipse data decks, individual include files may also be parsed, but
```

Comment:
Instead of "data decks" what is ment here is instead "complete .DATA file".

Reply:
Reworded wrt complete/incomplete data decks
```
Instead of complete :term:`decks <deck>`, individual include files may also be parsed, but
```